step-templates/run-octopus-runbook.json

```diff
@@ -3,12 +3,14 @@
     "Name": "Run Octopus Deploy Runbook",
     "Description": "This step will kick off a runbook.  The runbook can exist in the same space and project, or it can exist on a different instance altogether.  \n\n**Please Note**: Prompted variable values have to be text or sensitive variables.  Variable types such as AWS or Azure accounts will not work.\n\nThis step should be called from a worker machine.  If it is called from a target and the runbook runs on the same target you run the risk of a deadlock.\n\n",
     "ActionType": "Octopus.Script",
-    "Version": 15,
+    "Version": 16,
     "Author": "bobjwalker",
+    "CommunityActionTemplateId": null,
```

Comment:
```suggestion
```
Apologies, this line should be removed.
We will update export to exclude this in the future.